upgrade_test.py

```diff
@@ -1438,6 +1442,38 @@ def _update_argus_upgraded_version(self, node, new_version):
         except Exception as exc:  # pylint: disable=broad-except
             self.log.exception("Failed to save upgraded Scylla version in Argus", exc_info=exc)
 
+    def validate_limited_voters_feature_enabled(self):
+        InfoEvent(message="Check that limited voters feature enabled after upgrade for all nodes")
+        max_voter_number = 5
+        feature_state_per_node = []
+        for node in self.db_cluster.nodes:
+            result = wait_for(func=self.db_cluster.is_features_enabled_on_node,
+                              timeout=60,
+                              step=1,
+                              text=f"Check feature enabled on node {node.name}",
+                              throw_exc=False,
+                              feature_list=["GROUP0_LIMITED_VOTERS"],
+                              node=node)
+            feature_state_per_node.append(result)
+        if all(not enabled for enabled in feature_state_per_node):
+            InfoEvent(message="Limited voters feature is not supported or enabled on all nodes").publish()
+            return
+        elif all(feature_state_per_node):
+            InfoEvent(message="Limited voters feature enabled on all nodes").publish()
+            num_of_nodes = len(self.db_cluster.nodes)
+            num_of_non_voters = len(self.db_cluster.nodes[0].raft.get_group0_non_voters())
+            if num_of_nodes <= max_voter_number and num_of_non_voters > 0:
+                InfoEvent(message=f'Number of voters: {num_of_nodes - num_of_non_voters} are less than expected: {max_voter_number}',
+                          severity=Severity.ERROR).publish()
+            elif num_of_nodes > max_voter_number and num_of_non_voters == 0:
```

Comment:
I think it is incorrect and should be (num_of_nodes - num_of_non_voters)>5

Reply:
if you mean use instead of both conditions, then we can loose situations when number of nodes in cluster 4 and num of non voters is 1 and 4 -1 > 5  is false and we got missconfiguration

if instead elif condition, then it could be used to be sure that we have not greater than 5 voters.
@temichus

Reply:
case: num_of_nodes=7, and 6 voters, 1 non voter

num_of_nodes > max_voter_number and num_of_non_voters == 0 will be false. 

but this is an error: "Number of voters is higher than expected"

Reply:
so should be 

num_of_nodes > max_voter_number and (num_of_nodes - num_of_non_voters)>5

Reply:
fixed